react/Text/Text.sketch.js

```diff
@@ -33,11 +37,24 @@ export const text = {
   [`${symbolNames.heading}`]: <Text heading>Heading text</Text>,
   [`${symbolNames.subheading}`]: <Text subheading>Subheading text</Text>,
   [`${symbolNames.standard}/1. Default`]: <Text>Standard text</Text>,
-  [`${symbolNames.standard}/2. Secondary`]: <Text secondary>Standard critical text</Text>,
-  [`${symbolNames.standard}/3. Strong`]: <Text strong>Standard strong text</Text>,
-  [`${symbolNames.standard}/4. Positive`]: <Text positive>Standard positive text</Text>,
-  [`${symbolNames.standard}/5. Critical`]: <Text critical>Standard critical text</Text>,
-  [`${symbolNames.standard}/6. Link`]: <Text><TextLink href="#">Standard link text</TextLink></Text>,
+  [`${symbolNames.standard}/2. Secondary`]: (
+    <Text secondary>Standard critical text</Text>
+  ),
+  [`${symbolNames.standard}/3. Strong`]: (
+    <Text strong>Standard strong text</Text>
+  ),
+  [`${symbolNames.standard}/4. Positive`]: (
+    <Text positive>Standard positive text</Text>
+  ),
+  [`${symbolNames.standard}/5. Critical`]: (
+    <Text critical>Standard critical text</Text>
+  ),
+  [`${symbolNames.standard}/6. Link`]: (
+    <Text>
+      <TextLink href="#">Standard link text</TextLink>
+    </Text>
+  ),
+  [`${symbolNames.large}`]: <Text large>Large text</Text>,
```

Comment:
This line needs to be moved above the `standard` definitions, or the numbers are out of order, since `large` appears before `standard` in the `sizes` array defined above.